### PR TITLE
docs: update minimal node version

### PR DESCRIPTION
**Pull request by mGGk-fr**
Update documentation for dropped Node14 support

### CODE REVIEW

File: versioned_docs/version-2.x/appendices/migration-guides/index.mdx [hunk 1]
```diff
@@ -14,6 +14,14 @@ import TabItem from "@theme/TabItem";
 
 ## `2.24.0` -> `2.25.0`
 
+### Drop support of Node 14
+
+In this version we've **dropped support for Node 14**. The minimum required
+version is now Node 16
+
+You need to check your project dependencies for compatibility with at lease Node
+16
```
Comment (real34) [inline versioned_docs/version-2.x/appendices/migration-guides/index.mdx:23]:
```suggestion
In this version we've **dropped support for Node 14** (<abbr title="End Of Life">EOL</abbr>: 2023-04-30). The minimum required
version is now Node 16.

You need to check your project dependencies for compatibility with at least Node
16.
```

**Review by real34 — APPROVED** [review]

File: versioned_docs/version-2.x/appendices/migration-guides/index.mdx [hunk 2]
```diff
@@ -14,6 +14,15 @@ import TabItem from "@theme/TabItem";
...
 ## `2.24.0` -> `2.25.0`
 
+### Drop support of Node 14
+
+In this version we've **dropped support for Node 14**
+(<abbr title="End Of Life">EOL</abbr>: 2023-04-30). The minimum required version
+is now Node 16.
+
+You need to check your project dependencies for compatibility with at least
+Node 16.
```
Comment (paul-vd) [inline versioned_docs/version-2.x/appendices/migration-guides/index.mdx:24]: It's not super clear as to what actions integrators should take to achieve this.

**Review by paul-vd — APPROVED** [review]

File: versioned_docs/version-2.x/appendices/migration-guides/index.mdx (hunk 2 above)
Comment (mGGk-fr) [reply versioned_docs/version-2.x/appendices/migration-guides/index.mdx]: Added instruction to check dependencies versions on npm website, but don't know what to add

Comment (mGGk-fr) [reply versioned_docs/version-2.x/appendices/migration-guides/index.mdx]: https://deploy-preview-695--heuristic-almeida-1a1f35.netlify.app/docs/2.x/appendices/migration-guides/#drop-support-of-node-14

**Review by Kagaminara — APPROVED** [review]

**Pull request merged**